dev/tasks/exec/tasks.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const gulp = require( 'gulp' );
+const gutil = require( 'gulp-util' );
+const minimist = require( 'minimist' );
+
+const exec = require( './tasks/exec' );
+const log = require( '../../lib/log' );
+
+module.exports = ( config ) => {
+	const ckeditor5Path = process.cwd();
+	const packageJSON = require( '../../../package.json' );
+
+	// Configure logging.
+	log.configure(
+		( msg ) => gutil.log( msg ),
+		( msg ) => gutil.log( gutil.colors.red( msg ) )
+	);
+
+	const tasks = {
+		execOnRepositories() {
+			// Omit `gulp exec` part of arguments
+			const options = minimist( process.argv.slice( 3 ), {
+				alias: { t: 'task' },
```

Comment:
Do we need aliases? The same with `-m|--message` in the commit function. Let's KISS and avoid those.